### ISSUE TITLE
quant_sitemap fatal error: ServiceNotFoundException for simple_sitemap.manager

### ISSUE THREAD

**Issue by kepol** [quantcdn/drupal#51]
Fatal error when trying to seed.

```
Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException: You have requested a non-existent service "simple_sitemap.manager". Did you mean one of these: "simple_sitemap.queue", "simple_sitemap.logger", "simple_sitemap.generator", "simple_sitemap.entity_manager", "simple_sitemap.commands"? in Drupal\Component\DependencyInjection\Container->get() (line 156 of /var/www/quantweb-drupal/web/core/lib/Drupal/Component/DependencyInjection/Container.php).
```

Also, the `getSitemapVariants` method doesn't exist anymore so need to update to use similar logic that's used in `simple_sitemap`.

**Comment by kepol** [comment]
I'm going to create a WIP PR for the fix but the tests also need updating.

**Comment by kepol** [comment]
Sitemaps were fixed in #188 so closing this.
